From d0372d849d52d5a56fecd96a694eb92fdae87c59 Mon Sep 17 00:00:00 2001
From: Arnaud Ferraris <arnaud.ferraris@collabora.com>
Date: Wed, 15 Apr 2020 15:06:22 +0200
Subject: [PATCH] src: fix typos

---
 src/osk-manager.c | 2 +-
 src/wifimanager.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/wifimanager.c b/src/wifimanager.c
index dde115c..0d322fc 100644
--- a/src/wifimanager.c
+++ b/src/wifimanager.c
@@ -633,7 +633,7 @@ phosh_wifi_manager_class_init (PhoshWifiManagerClass *klass)
   props[PHOSH_WIFI_MANAGER_PROP_ENABLED] =
     g_param_spec_boolean ("enabled",
                          "enabled",
-                         "Wether wifi is enabled and a wifi device is available",
+                         "Whether wifi is enabled and a wifi device is available",
                          FALSE,
                          G_PARAM_READABLE | G_PARAM_EXPLICIT_NOTIFY);
 
diff --git a/src/osk-manager.c b/src/osk-manager.c
index ebeb9ab..0af3b06 100644
--- a/src/osk-manager.c
+++ b/src/osk-manager.c
@@ -244,7 +244,7 @@ phosh_osk_manager_class_init (PhoshOskManagerClass *klass)
   props[PHOSH_OSK_MANAGER_PROP_VISIBLE] =
     g_param_spec_boolean ("visible",
                           "visible",
-                          "Wheter the OSK is currently visible",
+                          "Whether the OSK is currently visible",
                           FALSE,
                           G_PARAM_READWRITE | G_PARAM_EXPLICIT_NOTIFY | G_PARAM_STATIC_STRINGS);
   g_object_class_install_properties (object_class, PHOSH_OSK_MANAGER_PROP_LAST_PROP, props);
-- 
2.25.1
